config/config_test.go

```diff
@@ -0,0 +1,32 @@
+package config
+
+import (
+	"testing"
+	"time"
+
+	. "github.com/smartystreets/goconvey/convey"
+)
+
+func TestSpec(t *testing.T) {
```

Comment:
Potentially:
```suggestion
func TestConfig(t *testing.T) {
```
would be a more accurate name here?